config.schema.json

```diff
@@ -109,6 +109,14 @@
         "default": false,
         "description": "Example: 'Turn on the Charger' to begin charging the car"
       },
+      "chargingAmps": {
```

Comment:
Hm, I think this should probably be a boolean (default false) that controls whether or not the service is added, instead of an integer? Configuring an integer value here doesn't really do anything as far as I can tell. See `chargeLimit` in the schema.

Reply:
Updated. Thank you for explaining what this is for.